wdl2cwl/main.py

```diff
@@ -78,13 +80,14 @@ def get_command(
                 append_str = "$(inputs." + sub_str + ")"
 
             new_command = new_command + append_str
+
             index = end_index + 1
         else:
             if command[index] != "\\":
                 new_command += command[index]
                 index += 1
             else:
-                new_command += " "
+                # new_command += " "
                 index += 1
```

Comment:
I think this block is the problem, you're filtering out the trailing `\` which we want to keep
``` diff
@@ -83,12 +84,8 @@ def get_command(
 
             index = end_index + 1
         else:
-            if command[index] != "\\":
-                new_command += command[index]
-                index += 1
-            else:
-                # new_command += " "
-                index += 1
+            new_command += command[index]
+            index += 1
     return new_command
```